### PR TITLE
Merge Hyperdrive content into master

### PR DESCRIPTION
**Pull request by vapaunic**
This PR would merge the new hyper-parameter with Hyperdrive tuning code into the master.

### CODE REVIEW

File: hyperparameter_tuning/CNN/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Multi-step forecasting with recurrent neural network that generates vector output
```
Comment (pechyony) [inline hyperparameter_tuning/CNN/README.md]: replace "recurrent" with "convolutional"

File: hyperparameter_tuning/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+> Note: This material is developed on an [Azure Data Science Virtual Machine (Ubuntu)](https://docs.microsoft.com/en-us/azure/machine-learning/data-science-virtual-machine/overview). This machine is pre-installed with many development tools we use in this tutorial, such as Anaconda, Python, etc.
+
+### Run hyper-parameter tuning notebook
+
+Hyper-parameter tuning is done in [hyperparameter_tuning.ipynb](./hyperparameter_tuning.ipynb) notebook. This notebook is used to tune several approaches:
+- Feed-forward network multi-step multivariate approach - [ff_multistep_config.json](ff_multistep_config.json)
+- RNN multi-step approach - [rnn_multistep_config.json](rnn_multistep_config.json)
+- RNN teacher forcing approach - [rnn_teacher_forcing_config.json](rnn_teacher_forcing_config.json)
+- RNN encoder decoder approach - [rnn_encoder_decoder_config.json](rnn_encoder_decoder_config.json)
```
Comment (pechyony) [inline hyperparameter_tuning/README.md]: Add pointer to JSON file of CNN

**Review by pechyony — CHANGES_REQUESTED** [review]
I added a couple of minor comments. Also, could put into hyperparameter_tuning.ipynb results of runs of all cells. Other than that, PR looks great.

File: hyperparameter_tuning/CNN/README.md (hunk 1 above)
Comment (vapaunic) [reply hyperparameter_tuning/CNN/README.md]: Good catch, thanks Dmitry. I'll address these comments.

**Comment by vapaunic** [comment]
@pechyony , I addressed your comments:
- Fixed up CNN readme wording, and the config file link
- added cell outputs in hyperdrive notebook.

Let me know if you have further comments, @angusrtaylor , @yijingchen , otherwise I'll merge the PR.

**Pull request merged**